src/charts/stacked-bar.js

```diff
@@ -867,6 +873,21 @@ define(function(require){
             return this;
         };
 
+        /**
+        * Gets or Sets the hasReversedStacks property of the chart, it reverse the stacks order.
+         * @param  {boolean} _x Desired horizontal direction for the graph
+         * @return { hasReversedStacks | module} If it is horizontal or Bar Chart module to chain calls
```

Comment:
Instead of 
```
@return { hasReversedStacks | module} If it is horizontal or Bar Chart module to chain calls
```
I would stick with the format used in the rest of the file which is:
```
@return { hasReversedStacks | module } Current hasReversedStacks or Chart module to chain calls
```
It is good to mention that this returns the value that tells whether the stacks are in horizontal above `@param`

Reply:
Oops I missed that, I'll fix that